tools/scripts/run_protocol_tests.py

```diff
@@ -0,0 +1,149 @@
+﻿#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+"""
+A script to run protocol tests for AWS SDK for C++:
+Provided --build_dir, it runs each built protocol test found.
+For each test executed, a mock http server is launched in parallel.
+"""
+import argparse
+import os
+import shutil
+import signal
+import subprocess
+import time
+
+SCRIPTS_DIR = os.path.dirname(__file__)
+PROTO_TEST_MOCK_HANDLER = f"{SCRIPTS_DIR}/protocol_tests_mock_server/protocol_tests_mock_server.py"
+
+
+def parse_arguments() -> dict:
+    """Parse arguments and/or print help
+
+    :return: dict with parsed arguments
+    """
+    parser = argparse.ArgumentParser(description="Protocol tests driver for AWS-SDK-CPP")
+    parser.add_argument("--debug", action="store_true", help="Print additional information")
```

Comment:
tiny suggestion: parser.add_argument("--debug", action="store_true", default = False, help="Print additional information")

and remove logic later